### PR TITLE
trpl: Few minor fixes (wording, formatting)

### PR DESCRIPTION
**Pull request by JIghtuse**

### CODE REVIEW

File: src/doc/trpl/iterators.md [hunk 1]
```diff
@@ -42,12 +42,12 @@ loop is just a handy way to write this `loop`/`match`/`break` construct.
 `for` loops aren't the only thing that uses iterators, however. Writing your
 own iterator involves implementing the `Iterator` trait. While doing that is
 outside of the scope of this guide, Rust provides a number of useful iterators
-to accomplish various tasks. Before we talk about those, we should talk about a
-Rust anti-pattern. And that's using ranges like this.
+to accomplish various tasks. But first, few notes about limitations of ranges.
```
Comment (steveklabnik) [inline src/doc/trpl/iterators.md]: This should be "a few notes"

File: src/doc/trpl/iterators.md [hunk 2]
```diff
@@ -42,12 +42,12 @@ loop is just a handy way to write this `loop`/`match`/`break` construct.
...
 
-Yes, we just talked about how ranges are cool. But ranges are also very
-primitive. For example, if you needed to iterate over the contents of a vector,
-you may be tempted to write this:
+Ranges are very primitive, and we often can use better alternatives. Consider
+following Rust anti-pattern: using ranges to emulate a C-style for loop. Let's
```
Comment (steveklabnik) [inline src/doc/trpl/iterators.md]: for should be in graves, so it looks like code. and we use curly 's, you can copy-paste from elsewhere in that file

**Comment by steveklabnik** [comment]
Thanks a ton! Just a few nits

**Comment by JIghtuse** [comment]
Ok, amended them.

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**